website/screens/components/dropdown/code/DropdownCodePage.tsx

```diff
@@ -17,14 +18,25 @@ const sections = [
           <thead>
             <tr>
               <th>Name</th>
+              <th>Type</th>
+              <th>Description</th>
               <th>Default</th>
-              <HeaderDescriptionCell>Description</HeaderDescriptionCell>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <td>options: object[]</td>
-              <td></td>
+              <td>
+                <DxcFlex direction="column" gap="0.25rem" alignItems="baseline">
+                  <StatusTag status="Required">Required</StatusTag>options
+                </DxcFlex>
+              </td>
+              <td>
+                <TableCode>
+                  {
+                    "{ label?: string; icon: string | (React.ReactNode & React.SVGProps <SVGSVGElement>); value: string }[]"
+                  }
+                </TableCode>
```

Comment:
add ? to icon like this:
`"{ label?: string; icon?: string | (React.ReactNode & React.SVGProps <SVGSVGElement>); value: string }[]"`

Reply:
True, fixed!